### PR TITLE
Remove minigraph loading in script

### PR DESCRIPTION
**Pull request by wen587**
#### Why I did it
Minigraph will be deprecated in the future. So minigraph related reload should be deleted.
#### How I did it
Remove unused load_minigraph
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

**Comment by wen587** [comment]
In sonic-ztp repo, do we need to remove minigraph here?
https://github.com/Azure/sonic-ztp/blob/master/src/usr/lib/ztp/plugins/graphservice#L90

File: files/image_config/updategraph/updategraph [hunk 1]
```diff
@@ -17,13 +10,6 @@ fi
 
 . /etc/sonic/updategraph.conf
 
-if [ "$enabled" = "reload_only" ]; then
```
Comment (qiluo-msft) [inline files/image_config/updategraph/updategraph:20]: We can still keep the if statement, and echo an error message to stderr, and exit with non zero.

**Comment by qiluo-msft** [comment]
@abdosi Please review on multi-ASIC use case.

File: files/image_config/config-setup/config-setup [hunk 2]
```diff
@@ -329,13 +320,8 @@ do_config_migration()
...
         # Disable updategraph
         disable_updategraph
-    elif [ -r ${MINGRAPH_FILE} ]; then
-        echo "Use minigraph.xml from old system..."
-        reload_minigraph
-        # Disable updategraph
-        disable_updategraph
     else
-        echo "Didn't found neither config_db.json nor minigraph.xml ..."
+        echo "Didn't found neither config_db.json..."
```
Comment (qiluo-msft) [inline files/image_config/config-setup/config-setup:324]: >neither [](http://example.com/codeflow?start=27&length=7)

remove "neither"

**Review by yxieca — CHANGES_REQUESTED** [review]
By keeping config_db.json before rebooting into new image. This code path is already avoided. There is NO point of removing this code.

Risk to nightly tests needs to be assesses.

File: files/image_config/updategraph/updategraph [hunk 3]
```diff
@@ -18,10 +11,8 @@ fi
 . /etc/sonic/updategraph.conf
 
 if [ "$enabled" = "reload_only" ]; then
```
Comment (qiluo-msft) [inline files/image_config/updategraph/updategraph:13]: We can focus on this file in this PR. Which seems safe to merge.

**Comment by wen587** [comment]
@yxieca, coud you review again?

**Review by yxieca — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**